src/main/resources/application.yml

```diff
@@ -47,6 +48,8 @@ spring:
    rest:
       # this points at the local WireMock server in the test environment.
       iexBasePath: http://localhost:${wiremock.server.port}/
+      iexHistoricalPath: https://cloud.iexapis.com/stable
+      iexApiToken: pk_fde37fe1b888413ab706e1aab237aebd
```

Comment:
For tests, you could probably use something like `testToken` as your API token instead of your real one. Ideally, we wouldn't want API tokens anywhere in our actual code-base. The IEX docs even recommend

> Do not embed API keys directly in code. Instead of directly embedding API keys in your application’s code, put them in environment variables or in include files that are stored separately from the bulk of your code—outside the source repository of your application. Then, if you share your code, the API keys will not be included in the shared files.

There isn't much we can do for this project to resolve that but at least the test config can be a sample token instead of a real one. Additionally, this is a `Publishable` token so it is less important to keep secret than a `Secret` token or actual username/password credentials.